care/emr/management/commands/load_fixtures.py

```
@@ -592,3 +604,541 @@ def _create_device(
         device.updated_by = super_user
         device.save()
         return device
+
+    def _create_lab_definition_objects_for_facility(self, facility, user=None):  # noqa: PLR0915
```

Pylint (3.3.7)

[refactor] Too many local variables (49/15)

(R0914)